### ISSUE TITLE
Overwrite SSL certificates if settings are changed

### ISSUE THREAD

**Issue by Xilonz** [TypistTech/trellis-cloudflare-origin-ca#41]
## Submit a feature request or bug report

- [x] I've read the [code of conduct](https://github.com/TypistTech/trellis-cloudflare-origin-ca/blob/master/CODE_OF_CONDUCT.md)
- [x] This is a feature request
- [ ] This is a bug report
- [ ] This request isn't a duplicate of an [existing issue](https://github.com/TypistTech/trellis-cloudflare-origin-ca/issues)
- [x] I've read the project [readme](https://github.com/TypistTech/trellis-cloudflare-origin-ca) and followed them (if applicable)
- [x] I've read the [Trellis docs](https://roots.io/trellis/docs) and followed them (if applicable)
- [x] This is not a Trellis support request that should be posted on the [Roots Discourse](https://discourse.roots.io/c/trellis) forums or [Trellis' issue tracker](https://github.com/roots/trellis/issues)
- [x] This is not a Cloudflare support request that should be reported to [Cloudflare support team](https://support.cloudflare.com/)

---

**What is the current behavior?**

When adding domains or changing the site_hosts the cloudflare_origin_ca doesn't get updated because {site_key}.pem already exists.
`[FATAL] Certificate file "{site_key}.pem" already exists, use -overwrite to overwrite`
This results in cloudflare not adding the origin ca to additional domains or updating the hostnames.

**What is the expected or desired behavior?**

I think this behavior should be documented and/or be resolved. A solution would be to allowing to overwrite the certificate file name or passing -overwrite using an argument.

**Comment by TangRufus** [comment]
> I think this behavior should be documented

Agree. The readme need to mention about how to renew/re-generate SSL cert.

In the meantime, see https://github.com/TypistTech/trellis-cloudflare-origin-ca/issues/40#issuecomment-501214880
